app/viewmodels/tad/posttransition/ConsignmentItemViewModel.scala

```diff
@@ -79,6 +79,16 @@ object ConsignmentItemViewModel {
       declarationType = consignmentItem.declarationType.orElseBlank,
       countryOfDispatch = consignmentItem.countryOfDispatch.orElseBlank,
       countryOfDestination = consignmentItem.countryOfDestination.orElseBlank,
-      supplementaryUnits = "" // TODO - should these come from IE015?
+      supplementaryUnits = ie015.Consignment.HouseConsignment
+        .find(_.sequenceNumber == houseConsignment.sequenceNumber)
+        .flatMap {
+          hc =>
+            hc.ConsignmentItem.find(_.goodsItemNumber == consignmentItem.goodsItemNumber).flatMap {
+              ci =>
+                ci.Commodity.GoodsMeasure.flatMap(_.supplementaryUnits)
+            }
+        }
+        .map(_.asString)
+        .getOrElse("")
```

Comment:
Nice. Alternatively you could explore pooling all of the consignment items for all house consignments into one list and comparing by `declarationGoodsItemNumber`? I don't know if that might simplify it a bit?

NB
- `goodsItemNumber` - item sequence number unique to a particular house consignment (i.e. multiple items can have the same `goodsItemNumber`)
- `declarationGoodsItemNumber` - item sequence number unique to the entire consignment (i.e. no two items can have the same `declarationGoodsItemNumber`)